packages/ts.runtime/src/cli.ts

```diff
@@ -69,6 +69,7 @@ export function cli(params: {
         filename: fileName,
         outputFilename: outputFileName ?? "output-file-name.ts",
         text: v,
+        prependTsCode: false
```

Comment:
Could you please make this a command line argument?

Reply:
And add tests for the cli both with prepend and without? There should be a tsr.test.ts file